lib/pushmi_pullyu.rb

```diff
@@ -60,4 +60,24 @@ def self.logger
   def self.logger=(log)
     PushmiPullyu::Logging.logger = log
   end
+
+  def self.server_running=(boolean)
```

Comment:
Maybe `status` or something instead of `boolean`? That's like naming a number `int` 🙃